### PR TITLE
[docs] Issue Triaging and Reviewing PRs

### PR DESCRIPTION
**Pull request by Pal-Wrtites**
## Description
Related to #1754 

## Checklist

- [ ] Follows [official](https://github.com/arviz-devs/arviz/blob/main/CONTRIBUTING.md#pull-request-checklist) PR format
- [ ] Includes a sample plot to visually illustrate the changes (only for plot-related functions)
- [ ] New features are properly documented (with an example if appropriate)?
- [ ] Includes new or updated tests to cover the new feature
- [ ] Code style  correct (follows pylint and black guidelines)
- [ ] Changes are listed in [changelog](https://github.com/arviz-devs/arviz/blob/main/CHANGELOG.md#v0xx-unreleased)

### CODE REVIEW

**Comment by ahartikainen** [comment]
Hi, I think this should be an issue, not a PR.

File: doc/source/contributing/issue_triaging.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# **Issue Triaging**
```
Comment (OriolAbril) [inline doc/source/contributing/issue_triaging.md:1]:
```suggestion
# Issue Triaging
```

Formatting as title should be enough, no need for bold too

File: doc/source/contributing/issue_triaging.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+It is often intimidating to start contributing when you are new and don't know where to start. Or you would like to contribute but don't have the time to code but prefer helping out occasionally with issue triaging. 
+
+The first step for you to get started would be to make sure you subscribe and get notifications of any new issue our team runs into. You can then chose to work on it at the moment or come back to it later. Staying subscribed will ensure you have real time and updated data on where we are with the issue.
```
Comment (OriolAbril) [inline doc/source/contributing/issue_triaging.md:5]:
```suggestion
The first step for you to get started would be to make sure you subscribe and get notifications of any new issue our team runs into. You can then chose to work on it at the moment or come back to it later. Staying subscribed will ensure you are notified whenever issues are created or updated and can therefore see where your help might be needed. 
```

File: doc/source/contributing/issue_triaging.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# **Issue Triaging**
+
+It is often intimidating to start contributing when you are new and don't know where to start. Or you would like to contribute but don't have the time to code but prefer helping out occasionally with issue triaging. 
+
+The first step for you to get started would be to make sure you subscribe and get notifications of any new issue our team runs into. You can then chose to work on it at the moment or come back to it later. Staying subscribed will ensure you have real time and updated data on where we are with the issue. 
+
+## How to subscribe to issue
```
Comment (OriolAbril) [inline doc/source/contributing/issue_triaging.md:7]:
```suggestion
## How to subscribe to ArviZ issues
```

File: doc/source/contributing/issue_triaging.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# **Issue Triaging**
+
+It is often intimidating to start contributing when you are new and don't know where to start. Or you would like to contribute but don't have the time to code but prefer helping out occasionally with issue triaging. 
+
+The first step for you to get started would be to make sure you subscribe and get notifications of any new issue our team runs into. You can then chose to work on it at the moment or come back to it later. Staying subscribed will ensure you have real time and updated data on where we are with the issue. 
+
+## How to subscribe to issue 
+
+If you are not familiar on how to set up notifications on github, please check the following - [Setting Up Notifications on GitHub](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications).
```
Comment (OriolAbril) [inline doc/source/contributing/issue_triaging.md:9]: I would have the first link point to https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository directly so readers are directed to the most relevant page and section right away. After that we can add extra links with more context and general guidance like you already did on the next paragraph

File: doc/source/contributing/issue_triaging.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## How to subscribe to ArviZ issues
+
+If you are not familiar on how to set up notifications on github, please check the following - [Setting Up Notifications on GitHub](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository)
+Once you are set up, you can [view your subscriptions](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/viewing-your-subscriptions) and [manage your subscritions](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions) to ensure you are not being inundated with the volume and are getting notiifcations only to issues of your interest. ,
+
+## Contribution Guidelines
+
```
Comment (OriolAbril) [inline doc/source/contributing/issue_triaging.md:13]: The points below are good, and leaving the links to the examples can also be helpful (we might want to find some better examples in some cases though), but they are ideas and examples of things that are part of issue triaging. They are not exhaustive *nor they aim to be*.

Comment (Pal-Wrtites) [reply doc/source/contributing/issue_triaging.md:13]: Okay, so now I would need some better examples to incorporate into the text. Where and what should I be looking for?

Comment (OriolAbril) [reply doc/source/contributing/issue_triaging.md:13]: I would leave the examples on a second plane for now and focus on an overview/introduction about issue triaging before the bullet points. It needs to be crystal clear that the examples are orientative only and *not exhaustive*. We want to provide examples to help people get started and see clear tasks for them to tackle, but we don't want to constrain them to do only the tasks on these examples. 

One of the goals we have when writing these guides to attract new and more diverse contributors is incorporating people with different backgrounds to the team who will most probably teach us too about issue triaging either directly or after a while.

Comment (Pal-Wrtites) [reply doc/source/contributing/issue_triaging.md:13]: OKay, will work on it :)

**Pull request merged**